src/components/BaseTable/BaseTable.stories.js

```diff
@@ -0,0 +1,122 @@
+import { boolean, select, text } from '@storybook/addon-knobs';
+import BaseTable from './BaseTable.vue';
+import { extendedPalette } from '../../tailwind/theme';
+import TypeOverline from '../TypeOverline/TypeOverline.vue';
+import BaseDivider from '../BaseDivider/BaseDivider.vue';
+
+const tableHeaderData = ['Pet', 'Color', 'Hairiness', 'Friendly?']
+const tableData = [
+  ['Dog', 'Brown', 'Long-hared', 'Absolutely'],
+  ['Dog', 'Gray', 'Short-haired', 'Absolutely'],
+  ['Cat', 'White', 'Long-hared', 'Not even a little'],
+  ['Lizard', 'Green', 'Bald', 'Meh'],
+  ['Squirrel', 'Brown', 'Short-haired', 'Meh'],
+];
+const tableFooterData = ['Pet Footer', 'Color Footer', 'Hairiness Footer', 'Friendly Footer']
+
+export default {
+  title: 'BaseTable',
+  component: BaseTable,
+  parameters: {
+    componentSubtitle: 'Basic table component.',
+  },
+};
+
+export const Gallery = () => ({
+  components: { BaseTable, TypeOverline, BaseDivider},
+  template: `
+  <div>
+    <TypeOverline tag="h2" class="mb-2" variant="large">Default</TypeOverline>
+
+    <base-table>
+      <tr v-for="row in tableData" :key="row">
```

Comment:
```suggestion
      <tr v-for="row in tableData" :key="row">
```

You can't use non-primitives like arrays/objects as keys. I know what you're probably thinking - cool I'll just use index as the key. You don't want to do that either if you can avoid and the linter will complain about it. You can google as to why, but the tl;dr is that Vue (and React) use the key to identify reusable pieces of the DOM. So if for some reason you want to move the first row to the fifth row and Vue can identify that row by a unique key then it can just detach the element and insert it rather than rebuilding the entire table. I don't want to overthink this since it's a Storybook example, but maybe just have one dog row and use that first cell as the unique identifier key?

Reply:
:) yeah I agree. I wasn't going to go crazy because it was just a storybook, but honestly, I have no problem making the example stronger. Thanks for the details here!